examples/step-47/doc/results.dox

```diff
@@ -211,3 +211,81 @@ make sense:
+  @f{align*}{
+    \int_K v_h (\Delta^2 u_h)
+     &= \int_K (D^2 v_h) : (D^2 u_h)
+       + \int_{\partial K} v_h \frac{\partial (\Delta u_h)}{\partial \mathbf{n}}
+       -\int_{\partial K} (\nabla v_h) \cdot (\frac{\partial \nabla u_h}{\partial \mathbf{n}}).
+  @f}
+
+  Summing up over all cells $K \in  \mathbb{T}$,
+  @f{align*}{
+  \sum_{K \in \mathbb{T}} \int_{\partial K} v_h \frac{\partial (\Delta u_h)}{\partial \mathbf{n}} = 0,
+  @f}
+  and by the definition of jump over cells,
+  @f{align*}{
+  -\sum_{K \in \mathbb{T}} \int_{\partial K} (\nabla v_h) \cdot (\frac{\partial \nabla u_h}{\partial \mathbf{n}}) = -\sum_{e \in \mathbb{F}} \int_{e} \jump{\frac{\partial v_h}{\partial \mathbf{n}} (\frac{\partial^2 u_h}{\partial \mathbf{n^2}}).
+  @f}
+  We separate interior faces and boundary faces of the domain,
+  @f{align*}{
+  -\sum_{K \in \mathbb{T}} \int_{\partial K} (\nabla v_h) \cdot (\frac{\partial \nabla u_h}{\partial \mathbf{n}}) = -\sum_{e \in \mathbb{F}^i} \int_{e} \jump{\frac{\partial v_h}{\partial \mathbf{n}} (\frac{\partial^2 u_h}{\partial \mathbf{n^2}})
+  - \sum_{e \in \partial \Omega} \int_{e} \jump{\frac{\partial v_h}{\partial \mathbf{n}} h,
+  @f}
+  Where $\mathbb{F}^i$ is the set of interior faces.
+  This leads us to
+  @f{align*}{
+  \sum_{K \in \mathbb{T}} \int_K (D^2 v_h) : (D^2 u_h) \ dx - \sum_{e \in \mathbb{F}^i} \int_{e} \jump{\frac{\partial v_h}{\partial \mathbf{n}} (\frac{\partial^2 u_h}{\partial \mathbf{n^2}}) \ ds
+  = \sum_{K \in \mathbb{T}}\int_{K} v_h f  \ dx + \sum_{e\subset\partial\Omega} \int_{e} \jump{\frac{\partial v_h}{\partial \mathbf{n}} h \ ds.
+  @f}
+
+  In order to symmetrize and stabilize the discrete problem,
+  we add symmetrization and stabilization term.
+  We finally get the $C^0$ IP finite element scheme for the biharmonic equation:
+  find $u_h$ such that $u_h =g$ on $\partial \Omega$ and
+  @f{align*}{
+  \mathcal{A}(v_h,u_h)&=\mathcal{F}(v_h) \quad \text{holds for all test functions } v_h,
+  @f}
+  where
+  @f{align*}{
+  \mathcal{A}(v_h,u_h)&=\mathcal{F}(v_h) \quad \text{holds for all test functions } v_h,
+  @f}
+  where
+  @f{align*}{
+  \mathcal{A}(v_h,u_h):=&\sum_{K \in \mathbb{T}}\int_K D^2v_h:D^2u_h \ dx
+  \\
+  &
+   -\sum_{e \in \mathbb{F}^i} \int_{e}
+    \jump{\frac{\partial v_h}{\partial \mathbf n}}
+    \average{\frac{\partial^2 u_h}{\partial \mathbf n^2}} \ ds
+   -\sum_{e \in \mathbb{F}^i} \int_{e}
+   \average{\frac{\partial^2 v_h}{\partial \mathbf n^2}}
+   \jump{\frac{\partial u_h}{\partial \mathbf n}} \ ds
+  \\
+  &+ \sum_{e \in \mathbb{F}^i}
+   \frac{\gamma}{h_e}
+   \int_e
+   \jump{\frac{\partial v_h}{\partial \mathbf n}}
+   \jump{\frac{\partial u_h}{\partial \mathbf n}} \ ds,
+  @f}
+  and
+  @f{align*}{
+  \mathcal{F}(v_h)&:=\sum_{K \in \mathbb{T}}\int_{K} v_h f \ dx
+  -
+  \sum_{e\subset\partial\Omega}
+  \int_e \jump{\frac{\partial v_h}{\partial \mathbf n^2}} h \ ds.
+  @f}
```

Comment:
I think it might be worth stating what is different in this formulation from the "clamped" version considered in the rest of the program.